backend/clickhouse/migrations/000006_create_logs_new.up.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE IF NOT EXISTS logs_new
+(
+    Timestamp          DateTime,
+    UUID               UUID,
+    TraceId            String,
+    SpanId             String,
+    TraceFlags         UInt32,
+    SeverityText       LowCardinality(String),
+    SeverityNumber     Int32,
```

Comment:
if we filter on severity text, should we have an index for that as well?

Reply:
Good question. I think we should hold off on adding new indexes until we confirm it's needed. In the case of severity text, most people will be searching where "SeverityText=error" and presumably they'll be able to find N (where N is the limit) logs pretty quickly in whatever selected time frame they choose.

If we find it to be slow, we can add it later since adding an index is a less complicated operation.